models/balancer/ethereum/balancer_v2_ethereum_trades.sql

```diff
@@ -11,87 +11,94 @@
 
 {% set project_start_date = '2020-03-13' %}
 
-with v2 as (
-    select
-        '2' as version,
-        tokenOut as token_bought_address,
-        amountOut as token_bought_amount_raw,
-        tokenIn as token_sold_address,
-        amountIn as token_sold_amount_raw,
-        poolAddress as project_contract_address,
+WITH v2 AS (
+    SELECT
+        '2' AS version,
+        tokenout AS token_bought_address,
```

Comment:
For style guide I recommend we introduce _ as a delimiter for aliased fields.

Reply:
Not sure if we can do this from the linter side